### DISCUSSION TITLE
steam++可以在树莓派4B中运行吗

### DISCUSSION THREAD

**Discussion by duoduo6666** [BeyondDimension/SteamTools#712] [Q&A]
steam++可以在树莓派4B中运行吗

**Comment by rmbadmin** [comment]
目前我没有条件测试，所以结论是不知道，但只要能满足运行环境的话，应该是可以的。

**Comment by Mossimos** [comment]
树莓派 3B 测试是可以运行 （但是非常卡  经常闪退不知道4会不会这样）
下 Steam++_linux_arm_v2.6.1 这版本的
![QQ图片20211203135947](https://user-images.githubusercontent.com/26396340/144553634-faef75df-e2f9-44e0-8c64-c47a344da281.jpg)

**Reply by x0chaoren0s** [reply]
请问大佬怎么在树莓派3b安装成功的？我下载了'Steam  _linux_arm64_v2.6.9.deb'后不论是双击打开还是sudo dpkg都装不了，提示架构不
`dpkg: 处理归档 Steam  _linux_arm64_v2.6.9.deb (--install)时出错：`
` 软件包体系结构(arm64)与本机系统体系结构(armhf)不符`
`在处理时有错误发生：`
` Steam  _linux_arm64_v2.6.9.deb`

**Reply by marc-shen** [reply]
你这个是系统的架构不符合，你给树莓派烧一个arm64架构的系统就好了。不要用armf的系统
